internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_runtime.go

```diff
@@ -89,6 +89,7 @@ func (r *Runtime) CanApplyImmediate(cfg config.Provider) error {
 	// * .machine.controlplane
 	// * .machine.kubelet
 	// * .machine.kernel
+	// * .machine.registries (not that auth is not applied immediately, containerd limitation)
```

Comment:
```suggestion
	// * .machine.registries (note that auth is not applied immediately, containerd limitation)
```

we now have this list in the docs as well:
https://github.com/talos-systems/talos/blob/master/website/content/docs/v0.15/Guides/editing-machine-configuration.md?plain=1#L40

Reply:
I already updated that doc in this PR!

Reply:
ah, ok. Missed that, sorry :)